src/classes/model-node.js

```diff
@@ -149,7 +149,16 @@ const ModelNode = class {
             && !(fieldValue instanceof Array)
             && fieldValue !== null
           ) {
-            const parentModel = DataModelHelper.loadModel(fieldValue.type, this.options.version);
+            // allow for data using @type instead of type
+            // because it's standard JSON-LD even in OA world expected to use the type shortcut
+            const fieldValueType = (fieldValue.type || fieldValue['@type']);
+            let parentModel;
+            try {
+              parentModel = DataModelHelper.loadModel(fieldValueType, this.options.version);
+            } catch (e) {
```

Comment:
Not keen on silently catch all exceptions, especially if silently swallowing it. Is there any way to catch the specific exception you're having issues with here?

Reply:
Fair enough. The error types were all Error both in this library and the data-models one so have added a custom error to be thrown then trying to load a model using a bad (undefined or non-existant) model name instead. Any other errors are re-thrown.

Reply:
Much better 👍